src/request.jl

```diff
@@ -17,10 +17,23 @@ worker_pointer_type() = Ptr{worker_type()}
 
 worker_pointer() = Base.unsafe_convert(worker_pointer_type(), pointer_from_objref(WORKERS))
 
-function free_threads!(freed_threads)
+function free_threads!(freed_threads::U) where {U<:Unsigned}
   _atomic_or!(worker_pointer(), freed_threads)
   nothing
 end
+function free_threads!(freed_threads_tuple::NTuple{1, U}) where {U<:Unsigned}
+  _atomic_or!(worker_pointer(), freed_threads_tuple[1])
+  nothing
+end
+function free_threads!(freed_threads_tuple::NTuple{N, U}) where {N,U<:Unsigned}
```

Comment:
```suggestion
function free_threads!(freed_threads_tuple::Tuple{U, Vararg{U, N}}) where {N,U<:Unsigned}
```

If you want to add the `U<:Unsigned`, I suggest `::Tuple{U,Vararg{U,N}}` instead of `::NTuple{N,U}`. That'll fix the failing test.